app/views/users/edit.html.haml

```diff
@@ -12,10 +12,10 @@
   = label_tag :gender, 'Female'
 
   = label_tag(:weight, 'Weight')
-  = number_field_tag(:weight, 'weight')
+  = number_field_tag(:weight, 'weight', min: 1, required: true)
 
   = label_tag(:height, 'Height')
-  = number_field_tag(:height, 'height')
+  = number_field_tag(:height, 'height', min: 1, required: true)
```

Comment:
we can add a Cuke/Capybara scenario to test the sad path if the user does not enter their weight/height and it should reject the input!

Reply:
We're in the process of adding tests for the entire form (tech debt from iteration 1), merging this in will make it easier for whoever writes tests to write comprehensive tests for the entire form functionality.